src/Response.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Inertia;
+
+use Illuminate\Http\JsonResponse;
```

Comment:
This adds the `illuminate/http`  dependency

Reply:
Right now we only have 
https://github.com/inertiajs/inertia-laravel/blob/e69451817be2700c66657f0e07e41be412716a40/composer.json#L25-L26

Reply:
Hmm, but when use facades (which I do in this library), such as `Illuminate\Support\Facades\App` or `Illuminate\Support\Facades\View` or others, it technically requires additional dependencies regardless. Which has me wondering what the usefulness of listing these dependencies really is.

Reply:
Don't know how other packages handle this kind of things tbh...

Reply:
At Spatie we require every package we explicitly import, so in this case we'd require `illuminate/http` as a dependency.

There's also the nuclear option of just requiring `laravel/framework`, but that brings along other issues (I forgot which ones, but I remember regretting it at some point)

Reply:
It just makes *no sense* to me to require `illuminate/support`, and then use the facades available in that package, but then omit the underlining dependencies for those facades. Things would blow up. I still don't get the point.

Reply:
Because that's not our problem. Our code is only `illuminate/support`'s API. If `illuminate/support` is calling other things under the hood, it's their responsibility to require those packages.

Reply:
Right, and that makes sense. I just mean _practically_ speaking, what advantage is any of this? How does it actually help anything?

The only reason I've even been inclined to include these kind of dependencies in the past is for testing purposes, because in the actual app...the Laravel dependencies are there. And to be honest, for testing, I've used `orchestra/testbench`, which does require `laravel/framework`.

Reply:
Yeah your second paragraph is true. However, if you're testing the package without testbench, it makes sense to correctly declare the dependencies.